sdk/textanalytics/azure-ai-textanalytics/README.md

```diff
@@ -215,80 +215,59 @@ TextAnalyticsAsyncClient textAnalyticsClient = new TextAnalyticsClientBuilder()
+    System.out.printf("Recognized Categorized Entity: %s, Category: %s, SubCategory: %s, Score: %s.%n",
+        entity.getText(), entity.getCategory(), entity.getSubCategory(), entity.getScore());
 }
 ```
 
 ### Recognize PII (Personally Identifiable Information) entity
-<!-- embedme ./src/samples/java/com/azure/ai/textanalytics/ReadmeSamples.java#L105-L114 -->
+<!-- embedme ./src/samples/java/com/azure/ai/textanalytics/ReadmeSamples.java#L96-L100 -->
 ```java
 String text = "My SSN is 555-55-5555";
-
-for (PiiEntity entity : textAnalyticsClient.recognizePiiEntities(text).getEntities()) {
-    System.out.printf(
-        "Recognized PII Entity: %s, Category: %s, SubCategory: %s, Score: %s.%n",
-        entity.getText(),
-        entity.getCategory(),
-        entity.getSubCategory(),
-        entity.getScore());
+for (PiiEntity entity : textAnalyticsClient.recognizePiiEntities(text)) {
+    System.out.printf("Recognized PII Entity: %s, Category: %s, SubCategory: %s, Score: %s.%n",
+        entity.getText(), entity.getCategory(), entity.getSubCategory(), entity.getScore());
 }
 ```
 
 ### Recognize linked entity
-<!-- embedme ./src/samples/java/com/azure/ai/textanalytics/ReadmeSamples.java#L121-L128 -->
+<!-- embedme ./src/samples/java/com/azure/ai/textanalytics/ReadmeSamples.java#L107-L111 -->
 
 ```java
 String text = "Old Faithful is a geyser at Yellowstone Park.";
-
-for (LinkedEntity linkedEntity : textAnalyticsClient.recognizeLinkedEntities(text).getLinkedEntities()) {
+for (LinkedEntity linkedEntity : textAnalyticsClient.recognizeLinkedEntities(text)) {
     System.out.printf("Recognized Linked Entity: %s, Url: %s, Data Source: %s.%n",
-        linkedEntity.getName(),
-        linkedEntity.getUrl(),
-        linkedEntity.getDataSource());
+        linkedEntity.getName(), linkedEntity.getUrl(), linkedEntity.getDataSource());
 }
 ```
 ### Extract key phrases
-<!-- embedme ./src/samples/java/com/azure/ai/textanalytics/ReadmeSamples.java#L135-L139 -->
+<!-- embedme ./src/samples/java/com/azure/ai/textanalytics/ReadmeSamples.java#L118-L121 -->
 ```java
 String text = "My cat might need to see a veterinarian.";
-
-for (String keyPhrase : textAnalyticsClient.extractKeyPhrases(text).getKeyPhrases()) {
+for (String keyPhrase : textAnalyticsClient.extractKeyPhrases(text)) {
     System.out.printf("Recognized phrases: %s.%n", keyPhrase);
 }
 ```
 
 ### Analyze sentiment
-<!-- embedme ./src/samples/java/com/azure/ai/textanalytics/ReadmeSamples.java#L146-L152 -->
+<!-- embedme ./src/samples/java/com/azure/ai/textanalytics/ReadmeSamples.java#L128-L131 -->
 ```java
 String text = "The hotel was dark and unclean.";
-
-for (TextSentiment textSentiment : textAnalyticsClient.analyzeSentiment(text).getSentenceSentiments()) {
-    System.out.printf(
-        "Analyzed Sentence Sentiment class: %s.%n",
-        textSentiment.getTextSentimentClass());
+for (SentenceSentiment textSentiment : textAnalyticsClient.analyzeSentiment(text).getSentenceSentiments()) {
```

Comment:
this is interesting, because the example only has one sentence so the document and the sentence have the same results.
what do you think about printing the `DocumentSentiment.Sentiment` instead?
or add another sentence to the document and then print both the document sentiment and the sentences sentiments to users can see the differente

Reply:
Will add another input sentence.

Reply:
Now that there are 2 sentences you could also print `textAnalyticsClient.analyzeSentiment(text).getSentiment()`